Cargo.toml

```diff
@@ -48,6 +50,5 @@ optional = true
 version = "0.1.4"
 optional = true
 
-[build-dependencies]
-phf_generator = "0.7.4"
-phf_shared = "0.7.4"
+[build-dependencies.string_cache_codegen]
+version = "0.2.11"
```

Comment:
Yes, that line is noted in the first comment of this PR as part of the instructions for trying it out.

However, I believe that won't work for downloads from crates.io because of `exclude = ["string-cache-codegen/**/*"]`. I added this exclude line because I think string-cache should consume string-cache-codegen from crates.io (like anything else will need to).

I could
 - add `path =` to demonstrates the tests pass or
 - add `path =` and remove `exclude =` for the purposes of this PR, then make a followup PR once string-cache-codegen is published to crates.io undoing this